javascript/linked-list/index.js

```diff
@@ -104,6 +104,28 @@ class LinkedList {
     }
   }
 
+
+  kthFromEnd(k) {
+    if (k < 1) {
+      return 'Exception';
```

Comment:
When the prompt says to return an exception, it is actually asking for a `throw new Error('Your custom error message here')`